tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -226,6 +227,47 @@ export const createSpaceFromFolder = async ({
   return (await response.json()) as Space
 }
 
+export const createDocumentFromTemplateUsingContextmenu = async ({
+  page,
+  resource,
+  actionName
+}: {
+  page: Page
+  resource: string
+  actionName: string
+}): Promise<void> => {
+  await page.locator(util.format(resourceNameSelector, resource)).click({ button: 'right' })
+  await page.locator(util.format(contextMenuAction, actionName)).click()
+}
+
+export const createFileFromTemplate = async ({
+  page,
+  resource,
+  webOffice,
+  via
+}: {
+  page: Page
+  resource: string
+  webOffice: string
+  via: string
+}): Promise<void> => {
+  if (via.startsWith('sidebar')) {
+    await sidebar.open({ page, resource })
+    await sidebar.openPanel({ page, name: 'actions' })
+    await page
+      .locator(util.format(sideBarActionButton, `Create from template via ${webOffice}`))
```

Comment:
can we have a variable for this? you can then reuse it
```suggestion
      .locator(util.format(sideBarActionButton, menuItem))
```